### PR TITLE
Move approvers/maintainers for Java in the java repo

### PR DESCRIPTION
**Pull request by bogdandrutu**
https://github.com/open-telemetry/opentelemetry-java/pull/759

### CODE REVIEW

File: community-members.md [hunk 1]
```diff
@@ -42,19 +42,7 @@ Maintainers (i.e., the "Technical Committee" or "TC"):
...
-- [Pavol Loffay](https://github.com/pavolloffay), RedHat
-- [Yang Song](https://github.com/songy23), Google
-- [Tyler Benson](https://github.com/tylerbenson), DataDog
-- [Armin Ruech](https://github.com/arminru), Dynatrace
-- [John Watson](https://github.com/jkwatson), NewRelic
-
-Maintainers:
-- [Bogdan Drutu](https://github.com/BogdanDrutu), Google
-- [Carlos Alberto](https://github.com/carlosalberto), LightStep
+Find out active members - approvers and maintainers of Java SDK in [open-telemetry/opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java) README file.
```
Comment (bhs) [inline community-members.md]: nit: the boilerplate for Go, JS, Python, etc seems clearer (and is more grammatical – complete sentences, etc) than this one. Can you copy-pasta one of those instead?

Comment (bogdandrutu) [reply community-members.md]: sure, I copied the .net probably that needs to be updated, or we should just be consistent everywhere. Will do that soon.

File: community-members.md [hunk 2]
```diff
@@ -42,11 +42,15 @@ Maintainers (i.e., the "Technical Committee" or "TC"):
...
 
-Find out active members - approvers and maintainers of Java SDK in [open-telemetry/opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java) README file.
+Repo: [open-telemetry/opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java)
+
+The list of active members (both "approvers" and "maintainers") for the OpenTelemetry Java API and SDK can be found in the [open-telemetry/opentelemetry-java README file](https://github.com/open-telemetry/opentelemetry-java#owners).
 
 ## .NET
 
-Find out active members - approvers and maintainers of .NET SDK in [open-telemetry/opentelemetry-dotnet](https://github.com/open-telemetry/opentelemetry-dotnet) README file.
+Repo: [open-telemetry/opentelemetry-dotnet](https://github.com/open-telemetry/opentelemetry-dotnet)
```
Comment (bhs) [inline community-members.md]: thanks :)

Comment (bogdandrutu) [reply community-members.md]: you are welcome :)

**Pull request merged**